shoov/shoov.profile

```diff
@@ -35,6 +35,11 @@ function shoov_install_tasks() {
     'display' => FALSE,
   );
 
+  $tasks['shoov_setup_enable_mail_template'] = array(
```

Comment:
Since we enabling theme, not template - change the task name.
Something like:
`shoov_setup_enable_mail_template` => `shoov_setup_enable_mimemail_template_theme`